modules/ocf_kerberos/manifests/init.pp

```diff
@@ -61,4 +61,35 @@
     hour    => 4,
     require => File['/usr/local/sbin/kerberos-git-backup'];
   }
+
+  # Allow Kerberos
+  ocf::firewall::firewall46 {
+    '101 allow kerberos':
+      opts => {
+        chain  => 'PUPPET-INPUT',
+        proto  => 'tcp',
+        dport  => 88,
+        action => 'accept',
+      };
+  }
+  # Allow Kerberos Password
+  ocf::firewall::firewall46 {
+    '101 allow kpasswd':
+      opts => {
+        chain  => 'PUPPET-INPUT',
+        proto  => 'tcp',
+        dport  => 464,
+        action => 'accept',
+      };
+  }
+  # Allow Kerberos Admin
```

Comment:
Perhaps we can limit this rule to desktops, I see no reason why people on tsunami/werewolves/death should be able to use `kadmin`